client-data/tools/image/image.js

```diff
@@ -80,7 +89,7 @@
   function createImageElement(data) {
     var img = svg.getElementById(data.id) || Tools.createSVGElement("image");
     img.setAttribute("id", data.id);
-    img.setAttribute("href", data.src);
+    img.setAttribute("href", getAbsoluteImageUrl(data.src));
```

Comment:
this is complicated. Can we remove the src altogether ?

```suggestion
    img.setAttribute("href", "./images/" + data.id);
```

that would stress me less :)

Reply:
How would the browser know where to pull the image to display from?

Reply:
The browser would make a request to `boards/{boardname}/images/{image_id}`, the server would check that the image exists, and if so, serve it.

Reply:
Yours is a much better approach 😅 This has been updated.